back/src/Guide/Cli.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+-- | This module adds command line interface to run server.
+--
+-- See help with 'guide --help' to see available commands.
+module Guide.Cli
+       ( Command (..)
+       , parseCli
+       ) where
+
+
+import Imports
+
+import Data.Version (showVersion)
+import Development.GitRev (gitCommitDate, gitDirty, gitHash)
+import Options.Applicative (Parser, ParserInfo, command, execParser, fullDesc, help, helper, info,
+                            infoHeader, infoOption, long, metavar, progDesc, short, strOption,
+                            subparser, (<|>))
+import Options.Applicative.Help.Chunk (stringChunk)
+
+import Paths_guide (version)
+
+import qualified Data.Text as T
+
+
+-- | All available commands
+data Command
+    = Guide               -- ^ run server
```

Comment:
Why not call it `RunServer` then? Or `Start` or something, maybe `Start` is a bit better than `Run`.